sdk/storage/azure-storage-stress/src/main/java/com/azure/storage/stress/ContentInfo.java

```diff
@@ -0,0 +1,54 @@
+package com.azure.storage.stress;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.zip.CRC32;
+
+public class ContentInfo {
+    private final long length;
+    private final long crc;
+    private final ByteBuffer head;
+
+    public ContentInfo(long crc, long length, ByteBuffer head) {
+        this.crc = crc;
+        this.length = length;
+        this.head = head;
+    }
+
+    public static Mono<ContentInfo> fromFluxByteBuffer(Flux<ByteBuffer> data) {
```

Comment:
I'd look into changing this API to `fromBinaryData` and pass `BinaryData` into this method. This more closely aligns to the long term goals of all data is passed using `BinaryData`, and should allow for synchronous tests to be written without `Flux`.